gbus/deduplicator/implementation/tx.go

```diff
@@ -0,0 +1,138 @@
+package implementation
+
+import (
+	"database/sql"
+	"time"
+
+	"emperror.dev/errors"
+	"github.com/sirupsen/logrus"
+
+	"github.com/wework/grabbit/gbus"
+	"github.com/wework/grabbit/gbus/deduplicator"
+	"github.com/wework/grabbit/gbus/tx"
+)
+
+var _ deduplicator.Store = &deduper{}
+
+type deduper struct {
```

Comment:
nitpicking, the struct is named deduper which is the main struct here yet the file name is tx.go